### ISSUE TITLE
unable to understand 

### ISSUE THREAD

**Issue by likui01** [BindsNET/bindsnet#421]
HI, 
is it possible to explain a few steps in the example code? 

n_classes = 10
assignments = -torch.ones(n_neurons, device=device)
proportions = torch.zeros((n_neurons, n_classes), device=device)
rates = torch.zeros((n_neurons, n_classes), device=device)

what each statement present. 
How do we calculate the labels?
assignments, proportions, rates = assign_labels(
                spikes=spike_record,
                labels=label_tensor,
                n_labels=n_classes,
                rates=rates,
            )

            labels = []
kindly provide some explanation, thank you

**Comment by Hananel-Hazan** [comment]
Please take a look on the [code ](https://github.com/BindsNET/bindsnet/blob/e69b975c16016dd412617f11bd19f7889128bced/bindsnet/evaluation/evaluation.py#L8) and the accompany papers for more detail:


1. Lattice map spiking neural networks (LM-SNNs) for clustering and classifying image data - [link](https://link.springer.com/epdf/10.1007/s10472-019-09665-3?author_access_token=nHODEUqmcqNJhF0ccyI55fe4RwlQNchNByi7wbcMAY6vESsguFcNT3DZ0WNofNVLvNCx0FGbqFD3vhOSzVjcY_Tt_znw6Rs3K-ld61Xs31hIT5nU39nY3TcBLFUWLNQ_q206gM1XLy5fgmItMLlT9g%3D%3D) or (Arxiv)[https://arxiv.org/abs/1906.11826]
2.  Locally Connected Spiking Neural Networks for Unsupervised Feature Learning - [link](https://www.sciencedirect.com/science/article/pii/S0893608019302333) or [Arxiv](https://arxiv.org/abs/1904.06269)
